liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_note-001.sql

```diff
@@ -6,7 +6,6 @@ CREATE TABLE dbo.nrt_interview_note
 (
     interview_uid    bigint                                          NOT NULL,
     nbs_answer_uid   bigint                                          NULL,
```

Comment:
Is the answer_val baked into another table or not required?

Reply:
answer_val was just not required. A handful of the other columns are the elements parsed out of answer_val.